static/app/stories/storyBook.tsx

```diff
@@ -0,0 +1,62 @@
+import {Children, Fragment, ReactNode} from 'react';
+import styled from '@emotion/styled';
+import invariant from 'invariant';
+
+import SideBySide from 'sentry/components/stories/sideBySide';
+import {space} from 'sentry/styles/space';
+
+type RenderFn = () => ReactNode | ReactNode[];
+type StoryFn = ((storyRender: RenderFn) => void) &
```

Comment:
Is this right, `&`?

Reply:
I know eh, it took some head smacking to get that sorted. Was it worth it? probably not.

Reply:
I don't understand what this does then?

Reply:
All it's doing is letter you optionally pass in a name

Reply:
I think at one point I had a story where that was going to be nice, but i can't find it anymore.

Reply:
Shouldn't this just be 


```tsx
type StoryFn = 
 | (storyRender: RenderFn) => void
 | (storyName: string, storyRender: RenderFn) => void;
```

then?

Reply:
gotta ask typescript about it

Reply:
let me know what they say

Reply:
seems happy this way, note the "+1 overload":
<img width="755" alt="SCR-20230918-ocdx" src="https://github.com/getsentry/sentry/assets/187460/859993c3-1ac2-4f5d-b0a0-41d8cf2c02fe">
<img width="630" alt="SCR-20230918-ocew" src="https://github.com/getsentry/sentry/assets/187460/ef91c2eb-15b2-46a0-b313-45edd999fd68">

Reply:
if the type uses `|` then it's not happy, and changes the signature to have `arg0`, always expecting 2 args:
<img width="704" alt="SCR-20230918-ocnz" src="https://github.com/getsentry/sentry/assets/187460/abe1ad71-488a-41e7-aeed-52e92c29ae60">

Reply:
removed it anyway, because i don't have an example yet without a name. And it could be a hot-reloading issue as stories without names are added/removed during dev.